### PR TITLE
Properly error out when using an `.xcworkspace`

### PR DESCRIPTION
**Pull request by brentleyjones**
Currently not supported, but building previously would silently fail instead of erroring out.

### CODE REVIEW

**Review by mattrobmattrob — APPROVED** [review]
We've been doing this for ages. 😆

Is this to fix a particular issue?

File: xcodeproj/internal/bazel_integration_files/calculate_output_groups.py [hunk 1]
```diff
@@ -160,8 +163,20 @@ def _calculate_label_and_target_ids(
         )
         return scheme_labels_and_target_ids
 
+    if not labels_and_target_ids:
+        print(
+            f"""\
+error: Failed to determine labels and targets. Note, currently `.xcworkspace`s \
+currently aren't supported. Please make sure you are opening the generated \
```
Comment (luispadron) [inline xcodeproj/internal/bazel_integration_files/calculate_output_groups.py:170]:
```suggestion
aren't supported. Please make sure you are opening the generated \
```

Comment (thiagohmcruz) [inline xcodeproj/internal/bazel_integration_files/calculate_output_groups.py:170]:
```suggestion
error: Failed to determine labels and targets. Note, currently `.xcworkspace`s \
aren't supported. Please make sure you are opening the generated \
```

**Review by luispadron — APPROVED** [review]

File: xcodeproj/internal/bazel_integration_files/calculate_output_groups.py [hunk 2]
```diff
@@ -160,8 +163,20 @@ def _calculate_label_and_target_ids(
...
+`.xcodeproj` file bundle directly. If you are, and you still get this error, \
+then lease file a bug report here: \
```
Comment (thiagohmcruz) [inline xcodeproj/internal/bazel_integration_files/calculate_output_groups.py:172]:
```suggestion
then please file a bug report here: \
```

**Comment by brentleyjones** [comment]
> Is this to fix a particular issue?

Fixes an issue if you are using Cocoapods, and Xcode updates the Pods project PIF file last, causing it to get picked up by this script, resulting in the wrong (no) information found.

**Pull request merged**